### DISCUSSION TITLE
Url Always Reset to Domain After Search, Change Page or Filter Table

### DISCUSSION THREAD

**Discussion by rizkhal** [rappasoft/laravel-livewire-tables#583] [Q&A] [answered]
Hy @rappasoft, thanks for this great package..
I have problem when i filter, search or change page pagination, the url always reset to root url (domain)..
My current solution is add `protected $queryString = []` but this not keep the url when i reload the page..

I see the demo site and it working, the url not reset to root url when i change the page, search or filter the table.. I was wondering how is your nginx configuration, can you give me the configuration?

**Comment by khedira00** [comment]
Hello, I have the same problem!:((

**Reply by rappasoft** [reply]
The demo site is a just a regular Forge provisioned server. I would not set the query string to `[]` in your component because that would reset it every time it rerendered i'd imagine.

**Reply by rizkhal** [reply]
Yes that's right, I intentionally added query string `[]` because if not then every time I do a search, filter or sort column the url will be reset to the root url..

I've asked about this [here](https://github.com/livewire/livewire/discussions/3725#discussioncomment-1422629), there is a video demo that might help you to see where I went wrong..

Thanks for you reply sir

**Reply by rappasoft** [reply]
That is strange i've never seen it before. The only thing I can think of is trying to use Clockwork to trace where the redirect is coming from.

**Reply by rizkhal** [reply]
This issue only in production..
In local and ngrok this issue is nothing..

Do you think this issue must be convert to issue (?)

Thanks..

**Reply by rappasoft** [reply]
I don't think this issue is related to the package. I have it deployed on many production environments.

**Reply by rizkhal** [reply]
I'am so sorry..
I will trying to use `Clockwork` to find this issue, thanks for giving me clue about Clockwork.

I'am already trying to find this issue long time ago but still don't known what's wrong here.. 
We would be very grateful if you provide an comment this discussion after testing this with clockwork..

Thanks 🙏🏻

**Reply by khedira00** [reply]
@rizkhal did you solved the problem?

**Reply by rizkhal** [reply]
@khedira00 not yet, still using `$queryString = []` for now

**Comment by rizkhal** [comment, marked as answer]
refer from https://laravel.com/docs/8.x/deployment#nginx

i forgot to add 
```bash
add_header X-Frame-Options "SAMEORIGIN";
add_header X-Content-Type-Options "nosniff";
```

Thanks
